ROADMAP.md

```diff
@@ -0,0 +1,105 @@
+# AutoGluon Roadmap 2022
+
+In 2022, our top priorities are:
+
+- [ ] Windows OS Support
+- [ ] Python 3.9 Support
+- [ ] Time-Series Support
+- [ ] HuggingFace Integration
+- [ ] Improved Multi-modal Modeling
+- [ ] Advance SOTA in Stack Ensemble Research (Deeper Stack Ensembles)
+- [ ] Cloud Training & Deployment
+- [ ] Parallel Model Training
+- [ ] Parallel Hyperparameter Tuning
+- [ ] Distributed Model Training
+- [ ] Distributed Hyperparameter Tuning
+- [ ] Semi-supervised Learning
+- [ ] Enhanced Model Distillation
+- [ ] Online Inference Optimization
+- [ ] Meta-Learning
+- [ ] Improved Feature Type Inference
+- [ ] Co-variate Shift Detection
+- [ ] Co-variate Shift Correction
+- [ ] Exploratory Analysis
+- [ ] Model Interpretability
+- [ ] Model Uncertainty
```

Comment:
Probably also worth adding
- Model monitoring
- Calibration (conformal methods)
- Serving speed optimizations (this trade-off has come up a few times previously)
- Speed improvements for image and text tasks
- Semisupervised learning

Reply:
Good point, added:

```
- [ ] Model Monitoring
- [ ] Model Calibration (Conformal Methods)
- [ ] Image Model Inference Optimization
- [ ] Text Model Inference Optimization
```

Semi-supervised and online inference optimization are already listed.